web_src/js/index.js

```diff
@@ -582,6 +582,25 @@ function initInstall() {
   });
 }
 
+function initIssueComments() {
+  if ($('.repository.view.issue .comments').length === 0) return;
+
+  $(document).click((event) => {
```

Comment:
I think a delegated handler would be better, e.g. `$(document).on('click', '.comment', fn)`

Reply:
this does not work. should handle click outside of comment

Reply:
Yeah I misread the code, disregard.